index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
```

Comment:
Since this is the main section, it is better to use a main tag instead of a div tag, please change it

Reply:
- [ ] [REQUIRED] You may use some introductory phrases ✋ like (e.g: `I think it would be better if you do ....`). This way you will avoid the Imperative Way 👉 (e.g: giving Orders)